projects/api-design-guidelines/docs/introduction.md

```diff
@@ -1,3 +1,7 @@
+---
+stoplight-id: d6a8ba3f3c186
+---
```

Comment:
This already was the id of this page on docs.publiq.be, I just added it as a proof-of-concept to show that `yarn docs:lint:fix` does not alter it.